vizro-mcp/README.md

```diff
@@ -0,0 +1,193 @@
+# Vizro MCP server
+
+Vizro-MCP is a [Model Context Protocol (MCP)](https://modelcontextprotocol.io/) server, which works alongside a LLM to help you create Vizro dashboards and charts.
+
+## Features of Vizro-MCP
+
+Vizro-MCP provides tools and templates to create a functioning Vizro chart or dashboard step by step. Benefits include:
+
+✅ One consistent framework for charts and dashboards with one common design language. ✅ Validated config output that is readable and easy to alter or maintain. ✅ Live preview of the dashboard to iterate the design until the dashboard is perfect. ✅ Use of local or remote datasets simply by providing a path or URL.
```

Comment:
I see my attempt to space the lines using triple end spacing failed because the linter removed it. Do you want to go back to double spaced paragraphs here and below?